contracts/interfaces/IExtendedStakingPoolFactory.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: GPL-3.0-only
+
+pragma solidity >=0.5.0;
+
+import "./IStakingPoolFactory.sol";
+
+interface IExtendedStakingPoolFactory is IStakingPoolFactory {
```

Comment:
- Leave a comment explaining the reason of existence of this interface
- Consider importing it only in Cover and StakingProducts (the contracts that are actually using this "new" function) as otherwise it unnecessarily polutes the codebase with changes
- Optionally, consider replacing "extended" with "complete" as it doesn't actually extends but only adds what we initially missed.